src/utils/shorten-url.js

```diff
@@ -9,7 +9,7 @@ import url from 'url';
 
 export default function shortenUrl(urlToShorten: string): Promise<string> {
   let longUrl = urlToShorten;
-  if (!longUrl.startsWith('https://profile.firefox.com/')) {
+  if (!longUrl.startsWith('https://profiler.firefox.com/')) {
```

Comment:
So this didn't provoke any issue, just we were doing the change all the time.
Which makes me wonder if we shouldn't remove this check and always do it. Tell me what you think :)